sw/host/opentitanlib/src/util/usb.rs

```diff
@@ -29,11 +29,12 @@ impl UsbBackend {
         for device in rusb::devices().context("USB error")?.iter() {
             let descriptor = match device.device_descriptor() {
                 Ok(desc) => desc,
-                _ => {
+                Err(e) => {
                     deferred_log_messages.push(format!(
-                        "Could not read device descriptor for device at bus={} address={}",
+                        "Could not read device descriptor for device at bus={} address={}: {}",
```

Comment:
I'm a bit surprised that you're able to format the error messages with `{}`.  Typically, error types implement `Debug` and you'd want to write `{:?}`. (No change needed, but I am surprised).